nbodykit/algorithms/tests/test_conv_power.py

```diff
@@ -201,13 +313,13 @@ def test_with_zhist(comm):
 
     # add n(z) from randoms to the FKP source
     nofz = InterpolatedUnivariateSpline(zhist.bin_centers, zhist.nbar)
-    fkp['randoms/NZ'] = nofz(randoms['z'])
-    fkp['data/NZ'] = nofz(data['z'])
+    fkp['randoms']['NZ'] = nofz(randoms['z'])
```

Comment:
Does this mean the slash syntax is replaced by more brakets?

Reply:
Both are supported now and give identical results -- sometimes it feels easier to slice with the species name first when the species name is a variable